packages/@glimmerx/service/index.ts

```diff
@@ -1 +1,3 @@
 export { service } from './src/decorator';
+
+export default class Service {}
```

Comment:
Is the plan to add more things here? or is this class needed to match the default export from @ember/service?

Reply:
The class is just needed to match the default export for the time being. We can consider adding more in the future, but for now an empty class is probably fine, don't want to commit to more API space before we think it through.